gax-bom/pom.xml

```diff
@@ -84,6 +84,20 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
Do we still need this section if the default is skip?

Reply:
Ah yep, removing.